mlflow/cli.py

```diff
@@ -320,6 +320,25 @@ def _validate_static_prefix(ctx, param, value):  # pylint: disable=unused-argume
     "Default: Within file store, if a file:/ URI is provided. If a sql backend is"
     " used, then this option is required.",
 )
+@click.option(
+    "--serve-artifacts-opt",
```

Comment:
```suggestion
    "--serve-artifacts",
```

Can we remove `-opt`?

Reply:
It was just to prevent the linter warning for shadowing a function name but the -opt suffix really doesn't make much sense from an optional arg perspective. I've removed it and I'll push changes soon.